README.md

```diff
@@ -260,7 +260,7 @@ for troubleshooting:
 1. By passing `stackTraceCollectionConfig`
 to `withLeakTracking` or `enableLeakTracking`.
 
-<video src="md_images/collect_callstack.mov" controls="controls" playsinline="playsinline"></video>
+https://user-images.githubusercontent.com/12115586/208321882-ecb96152-3aa7-4671-800e-f2eb8c18149e.mov
```

Comment:
Does this automatically show in the markdown? or do we need to turn it into a clickable link?

Reply:
Original way did not work, but this does. and I do not know why. See here: https://github.com/dart-lang/leak_tracker/blob/e4edef3cfd431f8c438e6353194da0eba197e118/README.md